### PR TITLE
Added strict mode

### PR DESCRIPTION
**Pull request by zrnorth**

### CODE REVIEW

File: src/lib/cli.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ program
     '*Note* This adds complexity to the logs that are hard to decipher, but included if needed for debugging'
   )
   .option('-d, --devMode', 'Shortcut for running cucumber-js directly')
+  .option('--strict', 'Fail fast if a step is undefined')
```
Comment (guykisel) [inline src/lib/cli.js:30]: should we also support `-S` as cucumber-js does?

**Comment by guykisel** [comment]
:+1:

File: src/lib/cli.js (hunk 1 above)
Comment (midniteio) [reply src/lib/cli.js:30]: we already have a `-s` flag, feels confusing to have both `-s` and `-S`

**Comment by midniteio** [comment]
👍

File: src/lib/cli.js (hunk 1 above)
Comment (zrnorth) [reply src/lib/cli.js:30]: we didn't add it because of confusion with the '-s' (lowercase) above.

**Pull request merged**